pygame_gui/elements/ui_button.py

```diff
@@ -41,6 +41,7 @@ class UIButton(UIElement):
                                 unique event.
     :param visible: Whether the element is visible by default. Warning - container visibility may
                     override this.
+    :param command: Functions to be called when an event occurs.
```

Comment:
```suggestion
    :param command: Functions to be called when a button event occurs.
```
Just so we are distinguishing what class of events these functions are dealing with.